tasks/internal_reconcile_report_job/internal_reconcile_report_job.py

```diff
@@ -95,7 +95,7 @@ def query_db(
         )
 
         jobs = []
-        for sql_result in sql_results:
+        for sql_result in sql_results.mappings():
```

Comment:
Add this to internal_reconciliation_rdbms.py

Reply:
added